doc/reference/ru/operations/tips.rst

```diff
@@ -0,0 +1,141 @@
+
+Предпочитайте много серверов с локальными жесткими дисками вместо меньшего числа серверов с подключенными дисковыми полками.
+Но для хранения архивов с редкими запросами полки всё же подходят.
+
+
+RAID
+----
+
+При использовании HDD можно объединить их RAID-10, RAID-5, RAID-6 или RAID-50.
+Лучше использовать программный RAID в Linux (``mdadm``). Лучше не использовать LVM.
+При создании RAID-10, нужно выбрать ``far`` расположение.
+Если бюджет позволяет, лучше выбрать RAID-10.
+
+На более чем 4 дисках вместо RAID-5 нужно использовать RAID-6 (предпочтительнее) или RAID-50.
+При использовании RAID-5, RAID-6 или RAID-50, нужно всегда увеличивать stripe_cache_size, так как значение по умолчанию выбрано не самым удачным образом.
+
+    .. code-block:: bash
+
+    echo 4096 | sudo tee /sys/block/md2/md/stripe_cache_size
+
+Точное число стоит вычислять из числа устройств и размер блока по формуле: ``2 * num_devices * chunk_size_in_bytes / 4096``.
+
+Размер блока в 1024 Кб подходит для всех конфигураций RAID.
+Никогда не указывайте слишком маленький или слишком большой размер блока.
+
+
+На SSD можно использовать RAID-0.
+Вне зависимости от использования RAID, всегда используйте репликацию для безопасности данных.
+
+Включите NCQ с длинной очередью. Для HDD стоит выбрать планировщик CFQ, а для SSD — noop. Не стоит уменьшать настройку readahead.
+На HDD стоит включать write ahread.
+
+
+Файловая система
+----------------
+
+Ext4 — самый проверенный вариант, стоит указывать опции монтирования ``noatime,nobarrier``.
+XFS также подходит, но не так тщательно протестирована в сочитании с ClickHouse.
+Большинство других файловых систем также должны нормально работать. Файловые системы с отложенной аллокацией работают лучше.
+
+
+Ядро Linux
+----------
+
+Не используйте слишком старое ядро Linux. В 2015 году 3.18.19 — достаточно свежее.
+Рассмотрите возможность использования сборки ядра от Яндекса: https://github.com/yandex/smart — это дает прирост в производительности не менее 5%.
+
+
+Сеть
+----
+
+При использовании IPv6, стоит увеличить размер кеша маршрутов.
+Ядра Linux до 3.2 имели массу проблем в реализации IPv6.
+
+Предпочитайте как минимум 10 Гбит сеть. 1 Гбит также будет работать, но намного хуже для починки реплик с десятками терабайт данных или для обработки распределенных запросов с большим объемом промежуточных данных.
+
+
+ZooKeeper
+---------
+
+Вероятно вы уже используете ZooKeeper для других целей. Можно использовать ту же инсталляцию ZooKeeper, если она не сильно перегружена.
+
+Лучше использовать свежую версию ZooKeeper, как минимум 3.5. Версия в стабильных дистрибутивах Linux может быть устаревшей.
+
+С настройками по умолчанию, ZooKeeper является бомбой замедленного действия:
+
+    Сервер ZooKeeper не будет удалять файлы со старыми снепшоты и логами при использовании конфигурации по умолчанию (см. autopurge), это является ответственностью оператора.
+
+Эту бомбу нужно обезвредить.
```

Comment:
You could link https://github.com/yandex/ClickHouse/blob/master/doc/administration/zookeeper.md for further information about ZooKeeper configuration.